auditbeat/docs/fields.asciidoc

```diff
@@ -3451,7 +3451,7 @@ type: text
 
 Path to the file.
 
-*`file.path.raw`*::
```

Comment:
From my understanding this PR only contains non-breaking changes. I think this constitutes a breaking change.

Reply:
True, thanks